content/_data/upgrades/2-277-3.adoc

```diff
@@ -0,0 +1,7 @@
+[#winstone]
+==== Winstone 5.16 Upgrade
+
+link:https://issues.jenkins.io/browse/JENKINS-65280[JENKINS-65280]
+
+The embedded Jetty container has been updated from version 9.4.38 to version 9.4.39.
+For a detailed list of changes, see the link:https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.39.v20210325[Jetty 9.4.39 changelog].
```

Comment:
If we do not expect breaking changes, this should just say there are no changes requiring update notes. Otherwise this is just a second changelog.

If there are, this needs to be a lot more specific and have actionable advice for admins.

Reply:
Thanks.  Fixed in 7c19120e993e3b414d4587114e35eb6dda649df5